ci/jobs/pr.yml

```diff
@@ -0,0 +1,38 @@
+jobs:
+  - name: dataworks-repo-template-terraform
+    plan:
+      - get: dataworks-repo-template-terraform-pr
+        trigger: true
+        version: every
+      - put: dataworks-repo-template-terraform-pr
+        params:
+          path: dataworks-repo-template-terraform-pr
+          status: pending
+        input_mapping:
+          dataworks-repo-template-terraform: dataworks-repo-template-terraform-pr
+      - task: basic-git-test
```

Comment:
as this is the terrraform template repo, it would make more sense to put the terraform plan in here

Reply:
But that would require terraform code to function.  This will suffice for an empty repo, until the point the user has some .tf to process